advocacy_docs/partner_docs/VisionXGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for VisionX'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      | SIB Visions       |
+| **Partner Product**   |  VisionX       |
+| **Web Site**      |    https://visionx.sibvisions.com/   |
+| **Version**   |  VisionX 5.6.925      |
+| **Product Description**      |  VisionX is a low-code platform for creating web and mobile applications extremely quickly and easily. VisionX and its Oracle Forms Migration Extension are the most efficient solution for semi-automated migration of Oracle Forms and Oracle APEX into modern web and mobile applications. VisionX, together with EDB Postgres Advanced Server migration, can help provide an exit strategy from Oracle to Postgres.    |
```

Comment:
Solution summary:
VisionX is a low-code platform for creating web and mobile applications. VisionX and its Oracle Forms Migration Extension are the an efficient solution for semi-automated migration of Oracle Forms and Oracle APEX into modern web and mobile applications. VisionX, together with an EDB Postgres Advanced Server migration, can help provide an exit strategy from Oracle to Postgres.